packages/governance/src/paramGovernance/paramMakers.js

```diff
@@ -2,18 +2,30 @@
 
 import { ParamType } from './paramManager.js';
 
+/**
+ * @param {bigint} value
+ */
```

Comment:
In the interest of vertical code density, for single-parameter functions, I usually do:

```suggestion
/** @param {bigint} value */
```

This reduces consistency somewhat. And if the function later takes more than one parameter, there is some churn. So it's a matter of taste.